modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavascriptEs6AngularGenerator.java

```diff
@@ -0,0 +1,191 @@
+package io.swagger.codegen.languages;
```

Comment:
New commit removes my generator and adds a "useEs6" CLI option to the JavascriptClosureAngular generator. If set it will use my templates instead to keep things seperate.